ChangeLog

```diff
@@ -44,6 +44,10 @@ Changelog
   #724
   (Thibault Suzanne)
 
+- BatList: add `fold_map`
```

Comment:
It would be nice to have the type of `fold_map` in the Changelog, so that people understand what it does.